style/components/_footer.scss

```diff
@@ -28,4 +26,30 @@
       margin-right: 5px;
     }
   }
+
+  .proxy-status {
+    padding: 1px 6px;
+    display: inline;
+    float: right;
+
+    &.with-info {
+      cursor: pointer;
+    }
+
+    &.working i {
+      color: $grass-dark;
+    }
+
+    &.partial i {
+      color: white;
+    }
+
+    &.offline i {
+      color: $grapefruit-dark;
+    }
+  }
+}
+
+.footer > *:not(:first-child) {
```

Comment:
That can probably be optimized... wildcard aside, should at least remove the `:not(:first-child)` and use a second rule for `:last-child` to override the margin.

So that results in `.footer > *` and `.footer > *:last-child`